var/spack/repos/builtin/packages/vtk/package.py

```diff
@@ -24,11 +24,7 @@ class Vtk(CMakePackage):
 
     license("BSD-3-Clause")
 
-    version(
-        "9.3.1",
-        sha256="8354ec084ea0d2dc3d23dbe4243823c4bfc270382d0ce8d658939fd50061cab8",
-        preferred=True,
-    )
+    version("9.3.1", sha256="8354ec084ea0d2dc3d23dbe4243823c4bfc270382d0ce8d658939fd50061cab8")
```

Comment:
Info: Because there is no version marked with `preferred=True`, the highest version is automatically the default version. `preferred=True` does not need to be specified in this case.